// _@vuepress_core@1.0.2@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/dengxianghui/Desktop/vuepress/node_modules/_@vuepress_core@1.0.2@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-71fd5202",
    path: "/hooks/rule.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-71fd5202").then(next)
    },
  },
  {
    name: "v-cf0d1e74",
    path: "/hooks/introduction.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-cf0d1e74").then(next)
    },
  },
  {
    name: "v-d16bd022",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-d16bd022").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-2122c598",
    path: "/hooks/custom.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2122c598").then(next)
    },
  },
  {
    name: "v-6aa63d2c",
    path: "/hooks/useContext.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6aa63d2c").then(next)
    },
  },
  {
    name: "v-487a6468",
    path: "/hooks/useMemo.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-487a6468").then(next)
    },
  },
  {
    name: "v-47835c20",
    path: "/hooks/useReducer.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-47835c20").then(next)
    },
  },
  {
    name: "v-48c2b0a8",
    path: "/hooks/useEffect.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-48c2b0a8").then(next)
    },
  },
  {
    name: "v-45b2ebcc",
    path: "/hooks/yourHooks.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-45b2ebcc").then(next)
    },
  },
  {
    name: "v-699d7122",
    path: "/hooks/useRef.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-699d7122").then(next)
    },
  },
  {
    name: "v-a878fa34",
    path: "/hooks/useState.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-a878fa34").then(next)
    },
  },
  {
    name: "v-405ded2c",
    path: "/guide/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-405ded2c").then(next)
    },
  },
  {
    path: "/guide/index.html",
    redirect: "/guide/"
  },
  {
    path: "/pages/guide.html",
    redirect: "/guide/"
  },
  {
    name: "v-786c7f00",
    path: "/webpack/introduction.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-786c7f00").then(next)
    },
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
